extensions/blocks/podcast-player/templates/playlist-track.php

```diff
@@ -30,16 +32,27 @@
 
 <li
 	class="<?php echo esc_attr( trim( $class ) ); ?>"
+	[class]="'<?php echo esc_attr( trim( 'jetpack-podcast-player__track ' . $secondary_colors['class'] ) ); ?> ' + (currentTrack == <?php echo esc_attr( $index ); ?> ? 'is-active' : '')"
```

Comment:
I found this very difficult to read. I feel there would be value in splitting it up potentially with well named vars that are concat'd together later.

Reply:
it's also not feature complete, will need more classes. I'll definitely extract this one